app/views/two_factor_authentication/sms_opt_in/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= image_tag(asset_url('alert/warning-lg.svg'), alt: '', width: 54, class: 'margin-bottom-2') %>
+
+<%= render PageHeadingComponent.new.with_content(t('two_factor_authentication.opt_in.title')) %>
+
+<p>
+  <%= t(
+        'two_factor_authentication.opt_in.opted_out_html',
+        phone_number: content_tag(:strong, @phone_configuration&.masked_phone),
+      ) %>
+</p>
+
+<%= link_to t('two_factor_authentication.mobile_terms_of_service'),
+            MarketingSite.messaging_practices_url,
+            class: 'margin-top-2' %>
+
+<%= button_to t('forms.buttons.send_security_code'),
+            login_two_factor_sms_opt_in_path,
+            class: 'usa-button usa-button--big margin-top-4' %>
```

Comment:
Can we confirm if this should be `--wide` as well, as are most of these sorts of primary action buttons?

cc @anniehirshman-gsa

Reply:
Yes - primary action buttons should be `--wide`, and default to full width on mobile. And 2.5 rem space on top and bottom. Thanks!

Reply:
Will update! A quick grep didn't help me, which classes do 2.5rem instead of 2rem spacing?

Reply:
Design system spacing units are such that `1 unit = 0.5rem`, so `margin-top-4 = margin-top: 2rem;` and `margin-top-5 = margin-top: 2.5rem`.

https://designsystem.digital.gov/design-tokens/spacing-units/

Reply:
updated in b1c8e13caacd174c5e7c84137116f3df14a8a8be